doc/data/messages/u/undefined-loop-variable/bad.py

```diff
@@ -0,0 +1,9 @@
+import random
+
+
+def iter_list(lst):
+    s = 0
+    for x in lst:
+        s += x
+
+    return s + x
```

Comment:
I think you added this file by mistake :)

Reply:
My bad, removed.